Userland/Libraries/LibJS/Bytecode/Pass/LoadElimination.cpp

```diff
@@ -29,7 +29,7 @@ static NonnullOwnPtr<BasicBlock> eliminate_loads(BasicBlock const& block, size_t
 
     auto new_block = BasicBlock::create(block.name(), block.size());
     HashMap<size_t, Register> identifier_table {};
-    HashMap<u32, Register> register_rerouting_table {};
+    static HashMap<u32, Register> register_rerouting_table {};
```

Comment:
This will need to be cleared when the executable changes, otherwise we get some really weird behavior

Reply:
Cleared as in resetting the hash map? Like clear all the previous data in it?

Reply:
Yes, as in resetting it, or calling `clear()` on it